namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/rds.tf

```diff
@@ -0,0 +1,32 @@
+module "rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.0"
```

Comment:
The RDS module is now on release 5.1 Please use that and make any changes necessary to your code.